src/sentry/quotas/redis.py

```diff
@@ -127,41 +141,54 @@ def get_quotas(self, project, key=None, keys=None):
 
         return results
 
-    def get_usage(self, organization_id, quotas, timestamp=None):
+    def get_usage(
+        self, organization_id: int, quotas: list[QuotaConfig], timestamp: float | None = None
+    ) -> list[int | None]:
         if timestamp is None:
             timestamp = time()
 
-        def get_usage_for_quota(client, quota):
+        def get_usage_for_quota(
+            client: RedisCluster, quota: QuotaConfig
+        ) -> tuple[str | None, str | None]:
             if not quota.should_track:
-                return (None, None)
+                return None, None
 
             key = self.__get_redis_key(
                 quota, timestamp, organization_id % quota.window, organization_id
             )
             refund_key = self.get_refunded_quota_key(key)
 
-            return (client.get(key), client.get(refund_key))
+            return client.get(key), client.get(refund_key)
 
-        def get_value_for_result(result, refund_result):
+        def get_value_for_result(result: Any, refund_result) -> int | None:
```

Comment:
I would leave `result` instead of doing `result: Any` -- the former helps us know we still need to add an annotation whereas the latter is easier to get lost

Reply:
Definitely makes sense